cmd/kops/create_cluster.go

```diff
@@ -361,6 +364,7 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringVar(&options.MasterTenancy, "master-tenancy", options.MasterTenancy, "The tenancy of the master group on AWS. Can either be default or dedicated.")
 	cmd.Flags().StringVar(&options.NodeTenancy, "node-tenancy", options.NodeTenancy, "The tenancy of the node group on AWS. Can be either default or dedicated.")
 
+	cmd.Flags().StringVar(&options.APILoadBalancerClass, "api-loadbalancer-class", options.APILoadBalancerClass, "Currently only supported in AWS. Sets the API loadbalancer class to eiether 'classic' or 'network'")
```

Comment:
To avoid this flag I'd argue for dropping `classic` and using `network`
NLB's have fixed IP addresses. Classic loadbalancers change IP addresses.
Where I am has restricted the API to known IP addresses.  Our VPN policy is to only send the required traffic through the VPN, rather than all traffic.
Our dev team still need  to interact with the clusters. We've been busy playing wack-a-mole with the classic IP's as they change.

Reply:
As mentioned in #8963 

> It would be lovely to use NLB instead of CLB, but there are some odd limitations on the > AWS side with how many NLBs that one can have per region (soft limit) and how many > NLB ENIs one can have per VPC (harder limit).